packages/www/hooks/use-api.tsx

```diff
@@ -600,7 +598,7 @@ const makeContext = (state: ApiState, setState) => {
           order,
           limit,
           cursor,
-          filters: f
+          filters: f,
```

Comment:
I think you might have autoformatted off of an old version of the repo, or Prettier - @adamsoffer and I reformatted everything to use their default settings while you were gone

Reply:
@iameli this change done according to the new formatting rules - I don't see what's the problem?